src/frontend/shortlist/src/views/PreferencesView.vue

```diff
@@ -0,0 +1,14 @@
+<script>
```

Comment:
Unsure about this file's presence here.

Reply:
I added it so that you and Daniel's previous work on preferences can be shown. It needs to be a url regardless, and in this case, it should be linked to the home page too.